providers/github/resources/github.go

```diff
@@ -31,7 +31,7 @@ func getUser(ctx context.Context, runtime *plugin.Runtime, conn *connection.Gith
 	if g.memoize == nil {
 		g.memoize = memoize.NewMemoizer(30*time.Minute, 1*time.Hour)
 	}
-	res, err, _ := g.memoize.Memoize("user", func() (interface{}, error) {
+	res, err, _ := g.memoize.Memoize("user"+user, func() (interface{}, error) {
```

Comment:
nit: 
```suggestion
	res, err, _ := g.memoize.Memoize("user-"+user, func() (interface{}, error) {
```
to help with readability (if we need to debug, etc)